Gemfile

```diff
@@ -2,7 +2,9 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "3.0.0.dev", :path => "."
+gem "logstash-core", "3.0.0.dev", :path => "./logstash-core"
+# gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
```

Comment:
this should be removed, it will be anyway once any bin/plugin command is executed